mullvad_module.sh

```diff
@@ -50,23 +50,23 @@ VPN_CODES=("${VPN_LOCATIONS[@]}")
 VPN_CODES+=("${COUNTRY_CODES[@]}")
 VPN_LOCATIONS+=("${COUNTRIES[@]}")
 
+MULLVAD_STATUS=$(mullvad status | cut -d' ' -f3)
```

Comment:
this call applies to the entire program, slightly reducing runtime by removing the functions needing to call `mullvad status` individually